README.md

```diff
@@ -390,6 +390,33 @@ will create a directory named `ruggery` and populate it with a working
 Rug archive project with a standard layout and TypeScript dependencies
 and a simple editor.  You can use the editors in this project to add
 more Rugs to the generated project.
+### AddLocalEditor
```

Comment:
Can you add a space above? Better yet, put it in alphabetical order in the README?